res/values/strings.xml

```diff
@@ -413,6 +425,9 @@
     <string name="pref_notifications">Notifications</string>
     <string name="pref_notifications_show">Show</string>
     <string name="pref_notifications_priority">Priority</string>
+    <string name="pref_notifications_explain">Enable system notifications on new messages</string>
+    <string name="pref_show_notification_content">Show message content in notification</string>
+    <string name="pref_show_notification_content_explain">Shows sender and first words of message content in notification</string>
```

Comment:
My gut feeling says that `notifications` would be better because you don't need an article for plurals (would be the same in German: `Absender und erste Wörter der Nachricht in Benachrichtigung anzeigen` sounds very choppy; you either need an article [`der Benachrichtigung`] or a plural [`Absender und erste Wörter der Nachricht in Benachrichtigungen anzeigen`])

Also I'd drop `content` as it adds no information.